modules/twinkleprod.js

```diff
@@ -185,7 +185,11 @@ Twinkle.prod.callbacks = {
 
 		// Remove tags that become superfluous with this action
 		text = text.replace(/\{\{\s*([Nn]ew unreviewed article|[Uu]nreviewed|[Uu]serspace draft)\s*(\|(?:\{\{[^{}]*\}\}|[^{}])*)?\}\}\s*/g, "");
-
+		if (mw.config.get('wgNamespaceNumber') === 6) {
+			// remove "move to Commons" tag - deletion-tagged files cannot be moved to Commons
+			text = text.replace(/\{\{(mtc|(copy |move )?to ?commons|move to wikimedia commons|copy to wikimedia commons)[^}]*\}\}/gi, "");
```

Comment:
One thing this misses is if there are spaces after `{{`. The regex otherwise seems fine but you might consider keeping the structure consistent with line 187. Frankly you could even merge this into line 187 entirely and remove the namespace check, as if for some reason there is a `{{move to commons}}` on an article then it obviously should also be removed :)